coffee/chosen.jquery.coffee

```diff
@@ -500,9 +500,8 @@ class Chosen extends AbstractChosen
     width = div.width() + 25
     div.remove()
 
-    container_width = @container.outerWidth()
-
-    width = Math.min(container_width - 10, width)
+    if @container.is(':visible')
+      width = Math.min(@container.outerWidth() - 10, width)
```

Comment:
@adunkman and I paired on this, because I couldn't reproduce the issue in my jquery HTML page.

While working on it, we discovered a fun issue with jquery's [outerWidth method](http://api.jquery.com/outerWidth/). The short of it is that if the element is hidden, one should not trust jquery to do anything sensible. For example, with a `style` of `width: 98%` (on a non-visible element), it will return `98`. _Le sigh_.

So we needed to do a different thing in the jQuery version than in the prototype version.

Go early 2000s!